src/data/faq.json

```diff
@@ -407,8 +407,8 @@
                             "Deleting the app should always be a last resort for solving one of the known or other issues with the app.",
                             "Since the app cannot delete the exposure logs by itself, whether they are cleared or not is a decision of the Exposure Notification Framework.",
                             "In the current version, deleting the app on iOS devices leaves the exposure log intact without any guarantees for the duration of storage.",
-                            "On Android devices, the log is deleted if Corona-Warn-App was the last app using the framework on the device. If you have SwissCovid or Immuni installed in addition to the Corona-Warn-App, the log will only be removed if these additional apps are de-installed or you manually trigger deletion of the exposure log.",
-                            "The way de-installation is handled can change without notice, The same applies to any changes made to the Corona-Warn-App itself. Therefore, only uninstall the app as a last resort to resolve an error.",
+                            "On Android devices, the current implementation <a href='https://developers.google.com/android/exposure-notifications/exposure-notifications-api#methods' target='_blank' rel='noopener noreferrer'>deletes the 'database and keys' from the device</a>. Hence, you should expect that the collected random IDs are deleted when the app is uninstalled.",
```

Comment:
To make it more in parallel to the German version, maybe:
On Android devices, the current implementation <a href='https://developers.google.com/android/exposure-notifications/exposure-notifications-api#methods' target='_blank' rel='noopener noreferrer'>deletes the 'database and keys' from the device</a> `when uninstalling the app`.

Reply:
or maybe even:
On Android devices, the current implementation `of the Exposure Notification Framework` [deletes the 'database and keys' from the device](https://developers.google.com/android/exposure-notifications/exposure-notifications-api#methods) `when uninstalling the app`.

Reply:
Take care: in my review the second line (411) seems to be deleted? That was not my intention, should stay in its place! sorry for any confusion.